src/chunk.c

```diff
@@ -40,7 +40,7 @@ Chunk_t *Uncompressed_SplitChunk(Chunk_t *chunk) {
     size_t curNumSamples = curChunk->num_samples - split;
 
     // create chunk and copy samples
-    Chunk *newChunk = Uncompressed_NewChunk(split * SAMPLE_SIZE);
+    Chunk *newChunk = Uncompressed_NewChunk(split * SAMPLE_SIZE, 0);
```

Comment:
@slice4e  I think it's better using here DC define:
https://github.com/RedisTimeSeries/RedisTimeSeries/blob/bf56f937f02165033bd1b7f32993ff340ad3ea0d/src/consts.h#L106